### ISSUE TITLE
Issue connecting to prod RavenDB in an AD domain

### ISSUE THREAD

**Issue by mauroservienti** [Particular/NServiceBus#2118]
Customer report - 5060,

Scenario:
- NServiceBus RavenDB is installed on a production server joined to an AD environment;
- NServiceBus Endpoint is installed on a different server jpoined to the same domain;
- the Endpoint is correctly configured to access RavenDB using a valid connection string in the config file;
- The Endpoint is using Windows Authentication to connect to RavenDB via a Domain account;

At endpoint startup hapèpens that the connection to RavenDB cannot be established, thus the endpoint fails to set timeout persistance, subscriptions and so on.

The first thing we tries is to move to the ApiKey auth and everything worked as expected stating the RavenDB wan correctly running and no third party issues were influencing the communication.

Digging deeper we discovered in the logs the following:

2014-05-07 12:58:52,246 [1] INFO    NServiceBus.Persistence.Raven.RavenUserInstaller [(null)] <(null)> - Adding user '<domain><usr>' to raven. Instance:'http://ip:port',   Database:'<db-name>'.
2014-05-07 12:58:52,261 [1] WARN  NServiceBus.Persistence.Raven.RavenUserInstaller [(null)] <(null)> - Failed to add user to raven. Processing will continue

basically the endpoint at startup tries to add a user that is already defined on the server and fails, I can repro on my side the issue using a server hosted on Azure, frem that point on each attempt to connect to RavenDB return a HTTP500.

I think that we should not try to create the RavenDB user.
